app/controllers/support/providers/locations/check_multiple_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Support
+  module Providers
+    module Locations
+      class CheckMultipleController < SupportController
+        include SuccessMessage
+
+        def show
+          school_details
+        end
+
+        def update
+          save
+          redirect_to support_recruitment_cycle_provider_locations_path
+        end
+
+        private
+
+        def provider
+          @provider ||= recruitment_cycle.providers.find(params[:provider_id])
+        end
+
+        def save
+          school_details.each(&:save)
```

Comment:
should this be a bang? what happens if one of them doesn't save?

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3420/commits/c65fc85f873fc57c27066a3ae82d1934f519d5bc